README.md

```diff
@@ -2,9 +2,10 @@
 
 This [Atlas App Services](https://cloud.mongodb.com) app is used to notify the community. The designated administrators can send messages to the community or a smaller test group. They can also add/remove other administrators.
 
-Community members sign up by sending "sub", "subscribe", or "add" to the SMS phone number and then confirm their number with the generated 7-digit code. 
```

Comment:
Commenting now

Reply:
What the hell am I supposed to do with a comment like that??

Reply:
More appropriate to validate that it's proper hipster and resolve?